app/scripts/controllers/courier-controller.js

```diff
@@ -13,8 +13,10 @@
     var courierVm = this;
     var stateRedirect = 'courier.orders';
     courierVm.createCourier = createCourier;
-    courierVm.courierForm = {};
     courierVm.messages = {};
+    courierVm.initCourier = initCourier;
+    courierVm.initCourier();
```

Comment:
Si, lo asigne asi para poder hacer el spec de esta función:
it('name and email should exist authenticated user', function () {
ctrl.initCourier();
expect($auth.user.name).to.exist; //jshint ignore:line
expect($auth.user.email).to.exist; //jshint ignore:line
});
No tengo mucho conocimiento en spec, por ello lo asigne asi. Por lo que si ubicaba 
de esta forma initCourier() en el controlador, en el spec no me reconocia la función.